airflow/sentry.py

```diff
@@ -150,13 +150,21 @@ def add_breadcrumbs(self, task_instance, session=None):
         def enrich_errors(self, func):
             """Wrap TaskInstance._run_raw_task to support task specific tags and breadcrumbs."""
 
+            session_args_idx = find_session_idx(func)
+
             @wraps(func)
-            def wrapper(task_instance, *args, session=None, **kwargs):
+            def wrapper(task_instance, *args, **kwargs):
                 # Wrapping the _run_raw_task function with push_scope to contain
                 # tags and breadcrumbs to a specific Task Instance
+
+                try:
+                    session = kwargs.get('session', args[session_args_idx])
+                except IndexError:
+                    session = None
```

Comment:
Is this the right behaviour, or is session a required for `add_breadcrumbs` to work?

Reply:
This behavior remains the same as before.

Reply:
Fair, point.